lib/service/customer_support_service.dart

```diff
@@ -498,4 +525,127 @@ class CustomerSupportServiceImpl extends CustomerSupportService {
       await createRenderingIssueReport(token, ["IPFS Loading"]);
     }
   }
+
+  @override
+  Future<List<Object>> getIssuesAndAnnouncement() async {
```

Comment:
I don't encourage using Object in development. Suggest creating an abstract class named `CustomerSupportThread` or some other name abstractable.